### PR TITLE
Feature/temp aoi dashboard

### PR DESCRIPTION
**Pull request by edbrett**
Screenshot 2019-08-27 at 18 20 32

Adds the pending message for dashboards for both public and private view. When a user clicks "got it!" the message disappears. 

Testing:
- Go to both a public and private area and check the messages are differnet
- Click to hide message

### CODE REVIEW

File: app/javascript/components/widgets/manifest.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ import * as treeCoverLocated from './widgets/land-cover/tree-cover-located';
 import * as emissions from './widgets/climate/emissions';
 import * as woodyBiomass from './widgets/climate/whrc-biomass/';
 import * as soilBiomass from './widgets/climate/soil-organic';
-import * as emissionsDeforestation from './widgets/climate/emissions-deforestation';
+// import * as emissionsDeforestation from './widgets/climate/emissions-deforestation';
```
Comment (blayhem) [inline app/javascript/components/widgets/manifest.js]: should we remove this comments?

File: app/javascript/pages/dashboards/components/pending-dashboard/component.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  render() {
+    const { className, isUserDashboard } = this.props;
+
+    return this.state.visible ? (
+      <div className={cx('c-pending-dashboard', className)}>
+        <div className="message">
+          <div>
+            <h3 className="title">
+              {isUserDashboard
+                ? 'Your custom dashboard is almost ready!'
```
Comment (blayhem) [inline app/javascript/pages/dashboards/components/pending-dashboard/component.jsx]: Nice touch

File: app/javascript/pages/failed-import/component.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, { PureComponent } from 'react';
+
+import Icon from 'components/ui/icon';
+import treeImage from 'assets/icons/error.svg';
+
+import './styles.scss';
+
+class ErrorPage extends PureComponent {
```
Comment (blayhem) [inline app/javascript/pages/failed-import/component.jsx]: Is this the old error page?

**Comment by blayhem** [comment]
Should the message disappear like the cookies one, after clicking 'Got it' the first time?

File: app/javascript/components/widgets/manifest.js (hunk 1 above)
Comment (edbrett) [reply app/javascript/components/widgets/manifest.js]: for now no. This is enabled in another branch.

File: app/javascript/pages/failed-import/component.jsx (hunk 3 above)
Comment (edbrett) [reply app/javascript/pages/failed-import/component.jsx]: yes. I am going to refactor to a reusable component shortly. For now its good enough I think

**Pull request merged**